blocks/library/image/style.scss

```diff
@@ -9,11 +9,19 @@
 	&[data-align="left"] {
 		float: left;
 		margin-right: $block-padding;
+
+		@include break-medium() {
+			margin-left: calc( 50% - #{ $visual-editor-max-width / 2 } );
+		}
```

Comment:
I'm asking about block authors too. I mean, should we move these styles outside the `image` block stylesheet and provide a way to reuse it across different blocks.